Sources/UInt128/UInt128.swift

```diff
@@ -67,11 +67,6 @@ public struct UInt128 {
         self.init(upperBits: 0, lowerBits: 0)
     }
 
-    public init(_ source: UInt128) {
-        self.init(upperBits: source.value.upperBits,
-                  lowerBits: source.value.lowerBits)
-    }
```

Comment:
Mind giving me an overview as to why you're removing this initializer? I remember it being a necessary part of functioning as a UIntX type way back when I wrote this. Has something changed since then? Also, how can we be sure that removing this won't break compatibility with existing code that depends upon this library?

Reply:
Addressed in https://github.com/Jitsusama/UInt128/pull/37#issuecomment-1032113218.